.github/workflows/run_tests.yml

```diff
@@ -35,12 +35,21 @@ jobs:
 
       - name: Install dependencies
         run: |
+          if [ "$RUNNER_OS" == "Windows" ]; then
+              pip install pywin32==306
+          fi
           pip install ecdsa==0.18.0 sympy==1.11.1
+        shell: bash
 
       - name: Build the project
         run: |
-          export CMAKE_OSX_ARCHITECTURES="arm64;x86_64"
-          ./build_extension.sh
+          if [ "$RUNNER_OS" == "Windows" ]; then
+              powershell.exe ./build_extension.ps1
+          else
+              export CMAKE_OSX_ARCHITECTURES="arm64;x86_64"
+              ./build_extension.sh
+          fi
+        shell: bash
```

Comment:
Let's rewrite the step to be runner specific, like here https://github.com/software-mansion-labs/crypto-cpp-py/blob/main/.github/workflows/package.yml#L23

Reply:
Having this in a single step made it clearer imo but okay